docs: align site with proxy-mode merge (#50)
tlsnotary/website: docs/faq.md

TLSNotary uses a multi-party computation (MPC) approach to secure the TLS session. Without MPC, the Prover would have full control over the TLS session keys and could forge the Server's responses. Zero-knowledge (ZK) proofs alone cannot prevent this. To prevent forged responses, the Verifier participates in the handshake, splitting the TLS session keys between the Prover and the Verifier.

[Proxy mode](/docs/protocol/proxy-mode) needs only ZK proofs: the Verifier proxies the connection with the Server, observes the encrypted traffic, and later verifies a ZK proof from the Prover that the plaintext is consistent with the encrypted session. This avoids MPC-TLS's bandwidth overhead, but introduces a network-path assumption between the Verifier and the Server.

MPC-TLS remains the default because its direct-connection model avoids that network assumption and provides stronger resistance to censorship. Proxy mode is the right choice when speed matters more than the strongest trust guarantees.

### Can I prove statements about TLS data in zero-knowledge with TLSNotary? Does it use zkSNARKs?

> Zero-knowledge functionality is provided by the higher-level `tlsn` crate built on top of the TLSNotary protocol.

Author: Hmm what does this sentence mean?
Maybe:
```suggestion
Zero-knowledge functionality will be provided by higher-level crates built on top of the TLSNotary protocol.
```
